examples/jdbc2-execute-operation/jdbc2_execute_operation.bal

```diff
@@ -0,0 +1,87 @@
+import ballerina/io;
+import ballerina/java.jdbc;
+import ballerina/sql;
+
+function initializeTable(jdbc:Client jdbcClient) returns int|string|sql:Error? {
+    //Execute drop table. The sql:ExecuteResult is returned during the successful execution.
```

Comment:
Add a space between // and Execute.
```suggestion
    // Execute drop table. The sql:ExecuteResult is returned during the successful execution.
```
Can you check other place as well?

Reply:
Included to add BBEs